torch/_dynamo/variables/builder.py

```diff
@@ -1285,6 +1285,7 @@ def _clone_input(value):
         proxy.node.meta["example_value"] = example_value
         return CUDAStreamVariable(proxy, example_value, **options)
     elif isinstance(example_value, int) and proxy.node.target in [
+        torch.sym_int,
```

Comment:
There's an argument to be had for just having `elif isinstance(example_value, primitive_types)` always return a constant here.

Reply:
We cannot easily do so, the graph breaks here are load bearing, see https://github.com/pytorch/pytorch/pull/103303